docs/content/installation/using-gcp-marketplace-package.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Using GCP Marketplace Ingress Controller
+description: "This document will walk you through the steps needed to deploy the NGINX Ingress Controller through the GCP Marketplace."
+weight: 2400
+doctypes: [""]
+toc: true
+docs: "DOCS-608"
+---
+
+This document will walk you through the steps needed to deploy and configure the NGINX Ingress Controller through the GCP Marketplace.
+
+## Product overview.
+
+The NGINX Ingress Controller an implementation of a Kubernetes Ingress Controller for NGINX and NGINX Plus.
```

Comment:
```suggestion
The F5 NGINX Ingress Controller an implementation of a Kubernetes Ingress Controller for NGINX and NGINX Plus.
```
We have some highly specific branding guidelines about the first instance of a product name on a page.

I will personally ensure pages are adherent to it, but figured I'd add this suggestion here since it'll be publicly facing imminently.